### ISSUE TITLE
Running cowrie gives error and exits

### ISSUE THREAD

**Issue by freekdk** [DShield-ISC/dshield#92]
I installed this on a Raspberry Pi 1B using openSUSE Tumbleweed with an adapted script install.sh. It works OK.
However after starting cowrie I have the following messages in log/cowrie.log:
# more /srv/cowrie/log/cowrie.log 
2017-11-03T01:01:20+0100 [-] Loaded output engine: jsonlog
2017-11-03T01:01:22+0100 [-] Loaded output engine: dshield
2017-11-03T01:01:22+0100 [-] Loaded output engine: mysql
2017-11-03T01:03:36+0100 [twisted.scripts._twistd_unix.UnixAppLogger#info] twistd 17.9.0 (/srv/cowrie/cowrie-env/bin/python2 2.7.13) 
starting up.
2017-11-03T01:03:36+0100 [twisted.scripts._twistd_unix.UnixAppLogger#info] reactor class: twisted.internet.epollreactor.EPollReactor.
2017-11-03T01:03:36+0100 [-] CowrieSSHFactory starting on 2222
2017-11-03T01:03:36+0100 [cowrie.ssh.factory.CowrieSSHFactory#info] Starting factory <cowrie.ssh.factory.CowrieSSHFactory instance at
 0xb5503058>
2017-11-03T01:03:37+0100 [-] Generating new RSA keypair...
2017-11-03T01:03:40+0100 [-] Generating new DSA keypair...
2017-11-03T01:03:43+0100 [-] Ready to accept SSH connections
2017-11-03T01:03:43+0100 [-] HoneyPotTelnetFactory starting on 2223
2017-11-03T01:03:43+0100 [cowrie.telnet.transport.HoneyPotTelnetFactory#info] Starting factory <cowrie.telnet.transport.HoneyPotTelne
tFactory instance at 0xb4ff4ee0>
2017-11-03T01:03:43+0100 [-] Ready to accept Telnet connections
2017-11-03T01:03:43+0100 [-] Received SIGTERM, shutting down.
2017-11-03T01:03:43+0100 [-] Received SIGTERM, shutting down.
2017-11-03T01:03:43+0100 [-] Unhandled Error
        Traceback (most recent call last):
          File "/srv/cowrie/cowrie-env/lib/python2.7/site-packages/twisted/application/app.py", line 396, in startReactor
            self.config, oldstdout, oldstderr, self.profiler, reactor)
          File "/srv/cowrie/cowrie-env/lib/python2.7/site-packages/twisted/application/app.py", line 311, in runReactorWithLogging
            reactor.run()
          File "/srv/cowrie/cowrie-env/lib/python2.7/site-packages/twisted/internet/base.py", line 1243, in run
            self.mainLoop()                                                                                                          
          File "/srv/cowrie/cowrie-env/lib/python2.7/site-packages/twisted/internet/base.py", line 1252, in mainLoop                 
            self.runUntilCurrent()
        --- <exception caught here> ---
          File "/srv/cowrie/cowrie-env/lib/python2.7/site-packages/twisted/internet/base.py", line 851, in runUntilCurrent
            f(*a, **kw)
          File "/srv/cowrie/cowrie-env/lib/python2.7/site-packages/twisted/internet/base.py", line 627, in stop
            "Can't stop reactor that isn't running.")
        twisted.internet.error.ReactorNotRunning: Can't stop reactor that isn't running.

2017-11-03T01:03:43+0100 [-] (TCP Port 2223 Closed)
2017-11-03T01:03:43+0100 [cowrie.telnet.transport.HoneyPotTelnetFactory#info] Stopping factory <cowrie.telnet.transport.HoneyPotTelne
tFactory instance at 0xb4ff4ee0>
2017-11-03T01:03:43+0100 [-] (TCP Port 2222 Closed)
2017-11-03T01:03:43+0100 [cowrie.ssh.factory.CowrieSSHFactory#info] Stopping factory <cowrie.ssh.factory.CowrieSSHFactory instance at
 0xb5503058>
2017-11-03T01:03:43+0100 [-] Main loop terminated.
2017-11-03T01:03:43+0100 [twisted.scripts._twistd_unix.UnixAppLogger#info] Server Shut Down.

The error message is obvious.

**Comment by freekdk** [comment]
Apparently I did something wrong in starting cowrie using "systemctl start cowrie.service" with my own version of cowrie.service.
I used the version of cowrie.service in the doc/systemd/ folder of the cowrie distribution, changed "home/cowrie" in "srv" and removed the hash (#) in front of Wants=syslog.target. Now I have a proper running cowrie.

The current one in production runs on a Raspberry Pi 3B. This one runs on a Raspberry Pi 1B.
